### PR TITLE
Update and lots of edits

### PR DESCRIPTION
**Pull request by ilario**
Here I edited pleeeeenty of things in the website.
Please tell me if I should break the PR in smaller ones.
What is missing is to update the *How it works* page to describe Babeld instead of BMX6, should I do it or wait for the next release?

### CODE REVIEW

File: development.txt [hunk 1]
```diff
@@ -136,5 +163,7 @@ Save and exit
 make -j$(nproc)
 --------------------------------------------------------------------------------
 
+Where you should replace +$(nproc)+ with the number of _physical_ CPU cores obtainable from +lscpu+ command as _Core(s) per socket_.
```
Comment (G10h4ck) [inline development.txt]: `$(nproc)` return the number of cores presents on the machine no need to do it manually

**Comment by G10h4ck** [comment]
To ease the review process it is better to split it in smaller PR

**Comment by nicopace** [comment]
> To ease the review process it is better to split it in smaller PR

Would support gio's request, as it is too big to review

File: development.txt (hunk 1 above)
Comment (ilario) [reply development.txt]: $(nproc) considers all the CPUs also due to hyperthreading, in my experience (with computational stuff just using CPU+RAM and nothing else, not with compilations, which could be different as it uses CPU+RAM+Disk+Network) is noticeably **faster** to use just as many threads as the **physical** cores.
I just tested on my laptop with a Intel i7-4600U @ 2.10GHz CPU (Thread(s) per core: 2, Core(s) per socket: 2) using buildroot and doing `make clean` + `make -j#`:
using 4 threads it took 14 min 22 s
using 2 threads it took 19 min 45 s
ok, so seems that with the kind of processes involved with OpenWrt compilation, the more processes the better, in the new PR I will remove this comment.

**Comment by ilario** [comment]
Closing this, I am creating a PR for each of the edited files, I still miss a few.
The content of the new PRs is not just this one broken apart but some further improvements are included.

**Comment by ilario** [comment]
All the changes have been ported to new pull requests, please review PR from #79 to #90.

**Pull request closed without merging**